e2e/sunshine.spec.ts

```diff
@@ -12,8 +12,103 @@ test.describe("Sunshine details page", () => {
       withFlag("/sunshine/operator/426/costs-and-tariffs", { sunshine: true })
     );
     await expect(resp?.status()).toEqual(200);
+    await sleep(3000);
+
+    await snapshot({
+      note: "Costs and tariffs - Network Costs",
+      locator: await page.getByTestId("details-page-content"),
+      fullPage: true,
+    });
+
+    const netTariffsButton = await page.getByTestId("net-tariffs-tab");
+    await expect(netTariffsButton).toBeVisible();
+    await netTariffsButton.click();
+    await sleep(3000);
+
+    await snapshot({
+      note: "Costs and tariffs - Net Tariffs",
+      locator: await page.getByTestId("details-page-content"),
+      fullPage: true,
+    });
+
+    const energyTariffsButton = await page.getByTestId("energy-tariffs-tab");
+    await expect(energyTariffsButton).toBeVisible();
+    await energyTariffsButton.click();
+    await sleep(3000);
+
+    await snapshot({
+      note: "Costs and tariffs - Energy Tariffs",
+      locator: await page.getByTestId("details-page-content"),
+      fullPage: true,
+    });
```

Comment:
Nit: I think it would be a better for maintenance to have

```javascript
for (const tab of [{ buttonLabel: 'costs and tariffs', screenshotLabel: 'Costs and tariffs' }, ...] = {
  await page.getByText(tab.buttonLabel)
  await page.waitForLoadState('networkidle')
      await snapshot({
      note: "Costs and tariffs - Energy Tariffs",
      locator: await page.getByTestId("details-page-content"),
      fullPage: true,
    });
}
```